xorg/keyboard-layout.txt

```diff
@@ -0,0 +1,64 @@
+SETTING THE KEYBOARD LAYOUT IN X
+________________________________________________________________________________
+
+
+Using command line or scripts
+________________________________________________________________________________
+
+Use setxkbmap to set keyboard layout preferences.
+
+* Build and install setxkbmap.
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ kiss b setxkbmap                                                         |
+|   $ kiss i setxkbmap                                                         |
+|                                                                              |
++------------------------------------------------------------------------------+
```

Comment:
This should match the style of the other pages. See https://k1ss.org/wiki/help/page-template